components/suggestion/at_mention_provider/at_mention_provider.jsx

```diff
@@ -68,6 +70,27 @@ export default class AtMentionProvider extends Provider {
         return profileSuggestions;
     }
 
+    // retrieves the parts of the group mention that should be checked
+    // against the term
+    getGroupSuggestions(group) {
+        const groupSuggestions = [];
+        if (!group) {
+            return groupSuggestions;
+        }
+
+        if (group.name) {
+            const groupnameSuggestions = getSuggestionsSplitByMultiple(group.name.toLowerCase(), Constants.AUTOCOMPLETE_SPLIT_CHARACTERS);
+            groupSuggestions.push(...groupnameSuggestions);
+        }
+        [group.display_name].forEach((property) => {
```

Comment:
I'm stumbling over this construct -- why doesn't this look like the `group.name` block above?